src/markdown/lists.scss

```diff
@@ -47,7 +47,7 @@
   }
 
   li {
-    word-wrap: break-all;
+    word-wrap: break-word;
```

Comment:
I think we can remove this line since `word-wrap: break-word;` should already get inherited from `.markdown-body`.

https://github.com/primer/css/blob/64a17a3bec490c35ac22fa33bffe21fe7433bce8/src/markdown/markdown-body.scss#L11

Or maybe the intention was to use `word-break: break-all;`. Not sure what's the exact difference?

Reply:
Good point, I missed that! Given that this CSS rule didn't have any effect in browsers, I think it's safe to remove this line as you've suggested 😄